scripts/termux-build-chroot.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+# Script for setting up a fake $PREFIX, on device, where packages can be
+# installed without modifying our normal $PREFIX on device. The installed
+# files can then be packaged into a .deb file. This script is just a simplified
+# version of the termux-chroot script.
+
+termux-build-chroot() {
+	SCRIPTNAME=termux-build-chroot
+
+	# For the /system/bin/linker(64) to be found:
+	ARGS="-b /system:/system"
+
+	# Bind $TERMUX_PKG_MASSAGEDIR/data to /data so that
+        # `make install DESTDIR=/data/data/com.termux/files/usr/` installs to
+        # this subdirectory in $TERMUX_PKG_MASSAGEDIR.
+	ARGS="$ARGS -b $TERMUX_PKG_MASSAGEDIR/data:/data"
+
+	# Keep home as it is. Packages shouldn't install files to it anyway
+	ARGS="$ARGS -b $HOME:$HOME"
+
+	# Mimic traditional Linux file system hierarchy - other Termux dirs:
+	for f in bin etc lib share tmp var; do
+		ARGS="$ARGS -b $PREFIX/$f:/$f"
+	done
+
+	# Mimic traditional Linux file system hierarchy- system dirs:
+	for f in dev proc; do
+		ARGS="$ARGS -b /$f:/$f"
+	done
+
+	# # Set $TERMUX_PKG_BUILDDIR as current directory:
+	# ARGS="$ARGS --cwd=$TERMUX_PKG_BUILDDIR"
+
+	# Root of the file system:
+	ARGS="$ARGS -r $(realpath $PREFIX/..)"
+
+	# Shell to execute:
+	PROGRAM=bash
+	if [ -x $HOME/.termux/shell ]; then
+		PROGRAM=`readlink -f $HOME/.termux/shell`
```

Comment:
Any point of using something other than `bash` for build chroot ?

Reply:
No, this is a leftover from the original termux-chroot script, fixed now